### ISSUE TITLE
Communication tool

### ISSUE THREAD

**Issue by lvillani** [just-install/just-install#236]
Hi there,

A while ago I set up a mailing list on Librelist (which seems to be down at the moment). It's clearly being unused and I think we could benefit either from switching to an IM solution or maybe switching to a mailing list hosted somewhere else (since Librelist seems to be having issues, at the time of writing this).

I would like to use something FOSS but I'm not opposed to stuff like Slack, Gitter, etc.

Since I already have two Slack accounts for work we could probably create and use a Slack channel (it's free), but I'd like to gather some input first.

The requirements are:
- Hosted (I don't really want to spend time doing sysadmin work)
- It should be possible to view previous history (this essentially rules out IRC without a bouncer) and/or offer searchable archives.

Cc: @guiweber @geek1011

**Comment by geek1011** [comment]
I would not like to use slack. It is a good tool, but I already have enough chat apps taking space and memory, but if you want to use it, it's fine with me. 

I already use Gitter, and I like it. IRCCloud seems interesting to me, and it fits all the requirements.

**Comment by lvillani** [comment]
I created an account on IRCCloud but the free plan makes it non-viable for me:

> Stay connected for 2 hours while inactive (permanent for first 7 days)

**Comment by lvillani** [comment]
I created a gitter account for our organization, may add a badge to our repos later on.

**Comment by Pafzedog** [comment]
Hello,

Association Framasoft is hosting a free Mattermost instance at https://framateam.org

I don't know if it will suit your needs but it is a FLOSS  alternative to Slack apparently .

**Comment by geek1011** [comment]
Framateam and mattermost seems good. It is in French though, but it is fine because I also understand French.

**Comment by Pafzedog** [comment]
You can change language in settings :) 
![language](http://pix.toile-libre.org/upload/original/1488491742.png)

**Comment by lvillani** [comment]
I can read a fair bit of French (cannot speak nor write though) :) Mattermost is great and we can probably reuse a great deal of Slack add-ons with it since the API is similar. Do newer versions of Mattermost support proper mobile push notifications?

Last time I tried one had to basically recompile their mobile apps to use one own's APNS/GCM server-side tokens and then distribute those apps. We ended up just using email notifications as poor-man's push notifications but it was really a tough sell overall.
